tests/core/test_configuration.py

```diff
@@ -1447,3 +1448,36 @@ def test_future_warning_only_for_code_ref(self, key):
             w = captured.pop()
             assert "your `conf.get*` call to use the new name" in str(w.message)
             assert w.category == FutureWarning
+
+
+def test_sensitive_values():
+    from airflow.settings import conf
+
+    # this list was hardcoded prior to 2.6.2
+    # included here to avoid regression in refactor
+    # inclusion of keys ending in "password" or "kwargs" is automated from 2.6.2
```

Comment:
Why kwargs?

Reply:
it bit us recently.  there are scenarios when users may supply sensitive info this way.  not a good practice but it happens apparently.

Reply:
basically ... erring on side of caution. mind you it's just the test to gather "suspected" sensitive. and we can ignore kwargs entries that are determined not sensitive.  essentially, forces a user to decide if item is sensitive when adding a kwargs config entry.